_episodes/conclusion.md

```diff
@@ -3,12 +3,117 @@ title: "Conclusion"
 teaching: 0
 exercises: 0
 questions:
-- "Key question (FIXME)"
+- "How can I assess the FAIRness of my data?"
+- "What do I do now to improve the FAIRness of my data?"
 objectives:
-- "First learning objective. (FIXME)"
+- "Relate each FAIRification practice with FAIR principles."
+- "Know about available tools/resources for assessing FAIRness of your data."
+- "Understand the next steps you can take to improve FAIRness of your data."
 keypoints:
-- "First key point. Brief Answer to questions. (FIXME)"
+- "FAIRification practices are about documentation, metadata, file formats, access to data,
+persistent identifiers, and data licenses."
+- "FAIR will change the way we do science."
+- "FAIR brings all the stakeholders together."
 ---
-FIXME
+
+## FAIRification practices and FAIR principles
+
+We discussed each FAIRification practice, i.e. documentation, metadata, file formats, access to data,
+persistent identifiers, and data licenses.
+We also explained what a FAIRification practice means in a FAIR context. In summary:
+
+> ## You make your research data findable by:
+>
+> ![findable]({{ page.root }}/fig/findable.jpg)
```

Comment:
I like the figures, but I'm not a huge fan of this layout. It would be nice if we could have the figure on the left, and then the points right next to them, but I guess that is difficult to do with the template. Howe about we just have a row of these images at the top or the bottom of this section?